documentation/E2E_TESTING.md

```diff
@@ -1,17 +1,28 @@
 # E2E Testing
 
+This End-2-End tests have been developed using Selenium. Selenium is a open-source automated testing framework used to validate web applications across different browsers and platforms. You can use multiple programming languages like create Selenium Test Scripts.
+
+For that reason we need an instance of Selenium running in the cluster, and then we can run our test using it.
+
 Inside the `tests/` folder in the repo there are two helm charts. The `e2e-test` helm chart and the `selenium-grid` helm chart.
 
 ## Usage
 
+### 1. Deploy Selenium
 To deploy the E2E tests it is important to have Selenium running and ready in the cluster first:
 
 ```shell
 cd tests/selenium-grid
 helm install --wait <my-selenium-release> . -n <PLATFORM_NAMESPACE>
 ```
 
-After this, you can run the e2e test the following way:
+### 2. Change `values.yaml`
+
+Change `values.yaml` file to adjust it to your deployment. (The most variables are fine to a default deployment)
```

Comment:
which values.yaml? From selenium chart? From the e2e chart¿

Reply:
take a look now @jtudelag, please